maas/client/utils/auth.py

```diff
@@ -67,7 +67,7 @@ def obtain_token(url, username, password, *, insecure=False):
 
         # Extract the CSRF token.
         login_doc = bs4.BeautifulSoup(response.content, "html.parser")
-        login_button = login_doc.find('input', value="Login")
+        login_button = login_doc.find('button', text="Login")
```

Comment:
No, but this needs to be rewritten in any case. This just unblocks anyone working with MAAS trunk, like me :)